### DISCUSSION TITLE
node ace configure @adonisjs/redis     Error

### DISCUSSION THREAD

**Discussion by maiconluizanschau** [adonisjs/core#4395] [Help]
I'm trying to run node ace configure @adonisjs/redis, but I only get this error
   Error: Package subpath './package.json' is not defined by "exports" in /node_modules/@adonisjs/lucid/package.json.
Node Version: v21.6.1
Yarn version: 1.22.21
Npm version: 10.2.4

**Comment by maiconluizanschau** [comment]
I'm trying to run node ace configure @adonisjs/lucid, but I only get this error

   Error: Package subpath './package.json' is not defined by "exports" in /node_modules/@adonisjs/lucid/package.json.
    node ace configure @adonisjs/lucid

**Comment by RomainLanz** [comment]
Hey @maiconluizanschau! 👋🏻 

A few things:

1. You are using an outdated version of `Yarn`, please upgrade or change your package manager;
2. What version of AdonisJS are you installing?

**Reply by maiconluizanschau** [reply]
adonis-ts-app@4.2.7
After several tests, the error that was occurring was in the lucid version, the latest version is not fully combatable.
I had to use npm i @adonisjs/lucid@18.4.0 and with that the project worked

**Reply by RomainLanz** [reply]
You are installing an AdonisJS 5 application; the latest version of all packages is for AdonisJS 6.

So yeah, upgrade your AdonisJS application or install an old version of the packages.

**Reply by iamNGT** [reply]
Hii @RomainLanz i have the same error but i have installed latest version of AdonisJS 6  and latest lucid please help

**Reply by thetutlage** [reply]
@iamNGT Can you please share the `package.json` file?

**Reply by iamNGT** [reply]
@thetutlage  here it is

{
  "name": "adonis_first_app",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "node ace serve --watch",
    "build": "node ace build --production",
    "start": "node server.js",
    "test": "node ace test"
  },
  "devDependencies": {
    "@adonisjs/assembler": "^5.9.6",
    "@japa/preset-adonis": "^1.2.0",
    "@japa/runner": "^2.5.1",
    "@types/proxy-addr": "^2.0.3",
    "@types/source-map-support": "^0.5.10",
    "adonis-preset-ts": "^2.1.0",
    "pino-pretty": "^10.3.1",
    "typescript": "~4.6",
    "youch": "^3.3.3",
    "youch-terminal": "^2.2.3"
  },
  "dependencies": {
    "@adonisjs/core": "^5.9.0",
    "@adonisjs/lucid": "^20.1.0",
    "@adonisjs/repl": "^3.1.11",
    "proxy-addr": "^2.0.7",
    "reflect-metadata": "^0.2.1",
    "source-map-support": "^0.5.21"
  }
}

**Reply by thetutlage** [reply]
Does not seem like a v6 app. The `@adonisjs/core` version is `5.9.0`

**Reply by iamNGT** [reply]
But i have installed the latest version using the command line from official docs, let me try again creating a new project

**Reply by thetutlage** [reply]
What command did you use?

**Reply by iamNGT** [reply]
sorry i used a wrong one, now everything work good thx.
i am new to adonisJs

**Comment by kev1nnnnnnn** [comment]
what fix?

**Comment by coreybrowndev** [comment]
Am I no longer able to use v5 for an app ? I must use v6 ?
